tools/msstitch/ready/msslookup_psms.xml

```diff
@@ -0,0 +1,54 @@
+<tool id="msslookup_psms" name="Create lookup table with PSMs">
+    <requirements>
+        <requirement type="package" version="2.2">msstitch</requirement>
+    </requirements>
+  
+    <description>necessary for protein groups and PSM quantification</description>
+    
+    <command>
+	    cp '$lookup' '$output' 
```

Comment:
CDATA and planemo will not lint this file I guess. description needs to come before requirements.